data-sources/mobile-dashboard-install/config.yml

```diff
@@ -0,0 +1,8 @@
+id: mobile-dashboard-install
```

Comment:
This is just speaking to a mobile dashboard, and resource is a general link to mobile monitoring. Should we opt instead to link to the a couple of our core mobile data sources (android, ios)?

Reply:
Lets just change the description to "Discover how to monitor your mobile app with New Relic"